sechub-doc/src/docs/asciidoc/documents/shared/springboot_security.adoc

```diff
@@ -28,15 +28,15 @@ Below is an example configuration with `Classic` mode and `OAuth2` in `JWT` mode
 sechub:
   security:
     server:
-      modes:
-        - oauth2
-        - classic
+      modes: oauth2, classic <1>
```

Comment:
I would prefer if blanks after the ',' would be omitted
in the documentation and examples.
```suggestion
      modes: oauth2,classic <1>
```